src/server/api/routers/dns-hole/router.ts

```diff
@@ -131,7 +133,13 @@ const processPiHole = async (app: ConfigAppType, enable: boolean) => {
 
 const collectPiHoleSummary = async (app: ConfigAppType) => {
   const piHole = new PiHoleClient(app.url, findAppProperty(app, 'apiKey'));
-  const summary = await piHole.getSummary();
+  const summary = await piHole.getSummary().catch(() => {
```

Comment:
Will there be any indication for the user to know, that something went wrong?
Also, always log errors when they happen!

Reply:
Yes errors still show in the browser log. I can add more logging if you want though

Reply:
Yes, please add server side logging too.

Reply:
Already does it actually